contracts/SnappAuction.sol

```diff
@@ -0,0 +1,81 @@
+pragma solidity ^0.5.0;
+
+import "./SnappBase.sol";
+
+
+contract SnappAuction is SnappBase {
```

Comment:
😍

Reply:
Although I'd still prefer if we could reuse code in solidity in better ways than inheritance.

Reply:
Is this solidity's problem, or something I should look into changing?

Reply:
solidity's problem